src/coktel.cpp

```diff
@@ -27,6 +27,7 @@
 #include <string.h>
 
 #include "coktel.h"
+#include "debug.h"
```

Comment:
Minor suggestion, include it only when DEBUG is defined.
```suggestion

#ifdef DEBUG
#include "debug.h"
#endif
```